packages/manager/modules/pci/src/projects/project/storages/containers/add/add.controller.js

```diff
@@ -318,6 +329,11 @@ export default class PciStoragesContainersAddController {
 
   onOfferSubmit() {
     this.displaySelectedOffer = true;
+    if (!this.isRightOffer()) {
+      this.isOffsiteReplicationEnabled = false;
+      this.forceEnableVersioning = false;
+      this.isBucketVersioningEnabled = false;
```

Comment:
There is three time the same in code maybe we can do a function
this.isOffsiteReplicationEnabled = false;
      this.forceEnableVersioning = false;
      this.isBucketVersioningEnabled = false;